tests/unit/utils/test_rsax931.py

```diff
@@ -276,3 +276,30 @@ def test_load_libcrypto(self):
             or hasattr(lib, "OPENSSL_init_crypto")
             or hasattr(lib, "OPENSSL_no_config")
         )
+
+    @patch.object(salt.utils.platform, "is_darwin", lambda: True)
```

Comment:
Let's move all these patch decorators to the test function both using `with` contexts please.